docs/_docs/tutorial/01-your-first-lte.md

```diff
@@ -131,41 +131,31 @@ After installing, you need to download the FPGA images packages by running _uhd
 $ sudo /usr/lib/uhd/utils/uhd_images_downloader.py
 ```
 
-#### 2. srsRAN
-
-On *Ubuntu 20.04(focal)*, one can install the required libraries with:
+The Laptop then needs to reboot, with the USRP unplugged. Only plug in the USRP after booting is done.
 
 ```bash
-$ sudo apt install cmake libfftw3-dev libmbedtls-dev libboost-program-options-dev libconfig++-dev libsctp-dev
+$ cd /usr/share/uhd/images/
+$ sudo uhd_usrp_probe
```

Comment:
After executing this, srsenb found the image without any help later when executing `sudo srsenb`